.generated-sources/blocklist-api/Cargo.toml

```diff
@@ -10,7 +10,6 @@ edition = "2021"
 serde = { version = "^1.0", features = ["derive"] }
 serde_with = { version = "^3.8", default-features = false, features = ["base64", "std", "macros"] }
 serde_json = "^1.0"
-serde_repr = "^0.1"
```

Comment:
Yeah I think one of my previous suggestions was that this should be built in a docker container which uses a specific version for everybody so that it's a deterministic build for everyone...